graphql/resolvers/editComment.test.js

```diff
@@ -0,0 +1,44 @@
+import { resolveRequestDocument } from 'graphql-request'
+import prismaMock from '../../__tests__/utils/prismaMock'
+import { editComment } from './editComment'
+
+const mockEditComment = {
+  id: 2,
+  content: 'i edited this comment'
+}
+
+describe('Edit comments resolver', () => {
+  test('should invoke prismaMock update', async () => {
+    prismaMock.comment.update.mockResolvedValue({
+      id: 2,
+      authorId: 1
+    })
+    prismaMock.comment.findUnique.mockResolvedValue({
+      authorId: 1
+    })
+    await expect(
```

Comment:
Is `await expect()` necessary? Wouldn't `expect(await editComment)` work?

Reply:
you are right. The await isn't necessary in these test cases. it's updated now